lib/msal-react/src/MsalProvider.tsx

```diff
@@ -31,10 +31,11 @@ type MsalState = {
     accounts: AccountInfo[];
 };
 
-enum MsalProviderActionType {
-    UNBLOCK_INPROGRESS = "UNBLOCK_INPROGRESS",
-    EVENT = "EVENT",
-}
+const MsalProviderActionType = {
```

Comment:
This can probably just be `const enum` as it's not exported.

Reply:
I would prefer to be on the safe side and use object literal instead of const enum in case it needs to be exported.

Reply:
This will never be exported as it is an implementation detail used for state management. I'll leave the final call up to you but please remove the export on the line below if you choose to keep it as an object literal.